README.md

```diff
@@ -1 +1,85 @@
-# Continents_Data
\ No newline at end of file
+# Continents_Data
+
+> Continents_Data is a a React and Redux SPA application.
+
+- In this project I used React and Redux to Fetch API data from Continents API.
+- You can filter through countries and briefly see all important countries data.
+
+## Built With
+
+- React
+- Redux
+- CSS
+
+## Dev Dependencies
+
+- React-routerDOM
+- Axios
+- Redux-toolkit
+- React-test-renderer
+- createAsyncThunk
+
+## Live Demo
+
+- - live demo: [Click here](https://continentsdata.netlify.app/)
+
+### Video Link
+
+- [Click here](https://www.loom.com/share/416d2612b52b4b85ad80e2f016ad6415)
```

Comment:
- [ ] Greate work on your video however you need to include some peace of code and show the different views (mobile...) of the app

Reply:
@Ismailco Thank you! I resolved it!